### PR TITLE
Add value text from the failure tag

### PR DESCRIPTION
**Pull request by BradLugo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -69,8 +69,9 @@ func convertJunitToSlack(junitFiles ...*junit.Suites) []slack.Attachment {
 			}
 
 			for _, result := range suite.Results {
+				failure := result.Failure
```
Comment (gavin-stackrox) [inline main.go:72]: It would be easier to read as a separate `resultToAttachment()`

File: main.go [hunk 2]
```diff
@@ -85,46 +86,88 @@ func convertJunitToSlack(junitFiles ...*junit.Suites) []slack.Attachment {
...
+						log.Printf("No junit failure message or value for %s", title)
+						continue
+					}
+
+					infoTextBlock := slack.NewTextBlockObject("mrkdwn", "*Info*", false, false)
+					infoSectionBlock := slack.NewSectionBlock(infoTextBlock, nil, nil)
+
+					if len(failureValue) > 3000 {
+						failureValue = failureValue[:3000]
+					}
```
Comment (gavin-stackrox) [inline main.go]: Already trimmed above?
```suggestion
```

File: main.go [hunk 3]
```diff
@@ -85,46 +86,88 @@ func convertJunitToSlack(junitFiles ...*junit.Suites) []slack.Attachment {
...
+				if len(failureValue) > 3000 {
+					failureValue = failureValue[:3000]
+				}
+
+				// Add some formatting to the failure title
+				failureTitleTextBlock := slack.NewTextBlockObject("plain_text", title, false, false)
+				failureTitleHeaderBlock := slack.NewHeaderBlock(failureTitleTextBlock)
+
+				// If there's no failure message or value, use a different message (this shouldn't be the usual case)
 				if failureMessage == "" {
```
Comment (gavin-stackrox) [inline main.go]: I think I would find this easier to read if it was:
```
If failureMessage == "" && failureValue == "" {
    handleNoFailureDetails()
}
if failureMessage != "" {
    addAttachmentFromFailureMessage()
}
if failureValue != "" {
    addAttachmentFromFailureValue()
}
```
Maybe addAttachmentFromFailureValue() addAttachmentFromFailureMessage() could be the same func.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]
I'm finding this hard to review as a big for loop. But that could be because it is the afternoon and I need coffee.

File: main.go (hunk 3 above)
Comment (BradLugo) [reply main.go]: Nah, it's not you. I wasn't a huge fan of it while writing it. You pointing it out proves it

File: main.go (hunk 1 above)
Comment (BradLugo) [reply main.go:72]: Separated out most of the logic below this into separate functions. Hopefully, it's a bit easier to read now. Feel free to add more suggestions regarding the readability

**Review by gavin-stackrox — APPROVED** [review]
Thanks, I can follow it now. LGTM!

**Pull request merged**